scripts/fetchParams.js

```diff
@@ -61,38 +61,2338 @@ const rdbToJSON = (rdb) => {
 
 let siteString = "";
 let countyString = "";
-let paramString = "";
 let agencyString = "";
 let locAquiferString = "";
 
 
 Promise.all(
   [get("https://help.waterdata.usgs.gov/code/site_tp_query?fmt=rdb"),          
   get("https://help.waterdata.usgs.gov/code/county_query?fmt=rdb"),
-  get("https://help.waterdata.usgs.gov/code/parameter_cd_query?fmt=rdb&group_cd=%"),
 get("https://help.waterdata.usgs.gov/code/agency_cd_query?fmt=rdb"),
 get("https://help.waterdata.usgs.gov/code/aqfr_cd_query?fmt=rdb")]
   ).then((value)=>{
 siteString = value[0];
 countyString = value[1];
-paramString = value[2];
-agencyString = value[3];
-locAquiferString = value[4];
+agencyString = value[2];
+locAquiferString = value[3];
  
-paramData = rdbToJSON(paramString)
-abridgedParamData = []
-paramData.forEach(value => {
-abridgedParamData.push({id: value["parm_cd"] , name: value["parm_nm"]});
-});
 
 
-
-let abridgedParamJSONString = JSON.stringify(abridgedParamData);
 let siteTypesJSONString = JSON.stringify(rdbToJSON(siteString));
 let countyJSONString = JSON.stringify(rdbToJSON(countyString));
 let agencyJSONString = JSON.stringify(rdbToJSON(agencyString));
 let locAquiferJSONString = JSON.stringify(rdbToJSON(locAquiferString));
 
+
+
+let IVParams = [
```

Comment:
You may want to include some information about where this data came from and how it was generated in a comment. I'm wondering if an approach using screenscrapping (i.e. parsing the HTML) would be more maintainable in the long run. Particularly, in dealing additions to the list and having a mechanism to reproduce the this object. 

Can the HTML change and the scripts require maintenance? Yes, but moving forward dealing with that maybe preferable to not being sure how this was created or how consistent it is with what is actually measured.